src/Buttons/Button.tsx

```diff
@@ -30,6 +30,8 @@ export interface ButtonProps extends MarkOneProps<HTMLButtonElement> {
   form?: string;
   /** Specifies the ref of the element */
   forwardRef?: Ref<HTMLButtonElement>;
+  /** Specifies the button type */
+  type?: string;
```

Comment:
What do you think about using `ButtonHTMLAttributes` to get better type safety here?


```suggestion
  type?: ButtonHTMLAttributes<HTMLButtonElement>['type'];
```

Reply:
That's a great point. I updated the typing in [3dac3f2](https://github.com/seas-computing/mark-one/pull/173/commits/3dac3f24ed30414ea27c2639c590cf4a3a64adaf).